R/cluster_sce.R

```diff
@@ -0,0 +1,27 @@
+#' Perform clustering on a SingleCellExperiment object
+#'
+#' @param sce SingleCellExperiment object that requires clustering
+#' @param pca_name The name used to access the PCA results
+#' @param BLUSPARAM A BlusterParam object specifying the clustering algorithm to
+#' use - can be `KmeansParam` or `NNGraphParam`
```

Comment:
We can be open to any kind of clustering here, if we want to be creative later.
```suggestion
#' @param BLUSPARAM A BlusterParam object specifying the clustering algorithm and parameters to use
```